### PR TITLE
Let tsfresh choose the value column if possible and increase test coverage

### PR DESCRIPTION
**Pull request by nils-braun**
Hi @hoesler!

Sorry for coming late (and even more sorry, because we needed long to review your PR). I realized that there was a small change concerning the API: before your very nice changes, it was possible to let tsfresh find out the column_value, if it was the only remaining column. I added this again.
Doing this, I tried to increase the test coverage a bit.

Feel free to drop a comment, if you want!

### CODE REVIEW

**Comment by hoesler** [comment]
Ni @nils-braun.

Apparently I missed that. Thanks!

But why did you change the `value_columns` arg of `WideTsData` from `list[str]` to `str`? If I just look at the class in isolation, it doesn't feel right to force users to select all or only one column. The current API is reflected in `to_tsdata`, so I would move more restricted selection logic to that function. Users can then circumvent that logic, by passing `WideTsData` directly instead of data frames.

**Comment by nils-braun** [comment]
I though that it makes the logic of the None-column_value easier, but you are right, in the end both is fine. 
I will change it back to your original propsal.

**Comment by nils-braun** [comment]
@hoesler Are you fine with merging this?

**Comment by nils-braun** [comment]
I assume you are :-) If not, just comment and we can change it back!

**Comment by hoesler** [comment]
Damn. I started a review a long while ago, but forgot to actually submit it. Sorry! Here are some small comments I had.

**Comment by nils-braun** [comment]
Ok, thanks! I will work on the issues and open a new PR!

File: tsfresh/feature_extraction/data.py [hunk 1]
```diff
@@ -164,17 +172,16 @@ def __init__(self, df, column_id, column_sort=None, value_columns=None):
         :type column_sort: str|None
 
         :param value_columns: list of column names to treat as time series values.
-            If `None`, all columns except `column_id` and `column_sort` will be used.
+            If `None` or empty, all columns except `column_id` and `column_sort` will be used.
```
[the remark this answers is not in the document]
Comment (nils-braun) [reply tsfresh/feature_extraction/data.py:175]: Nice catch! Thanks

**Pull request merged**